test/java/seedu/addressbook/commands/ViewCommandTest.java

```diff
@@ -0,0 +1,159 @@
+package seedu.addressbook.commands;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+import static seedu.addressbook.util.TestUtil.assertCommandResult;
+
+public class ViewCommandTest {
+    private AddressBook addressBook;
+    private AddressBook emptyAddressBook;
+    private List<ReadOnlyPerson> emptyDisplayList = new ArrayList<ReadOnlyPerson>();
+    private List<ReadOnlyPerson> listWithAll;
+    private List<ReadOnlyPerson> listWithSome;
+
+    @Before
+    public void setUp() throws Exception {
+        emptyAddressBook = TestUtil.createAddressBook();
+
+        TypicalPersons td = new TypicalPersons();
+        addressBook = td.getTypicalAddressBook();
+        listWithAll = Arrays.asList(td.getTypicalPersons());
+        listWithSome = Arrays.asList(td.amy, td.candy, td.dan);
+    }
+
+    @Test
+    public void viewCommand_invalidIndex_returnsInvalidIndexMessage() {
```

Comment:
@damithc Do you prefer the naming to be `execute_invalidIndex...` or `viewCommand_invalidIndex...`?

I see the `viewCommand_...` naming was introduced by #118 (my fault, I should have noticed it), which is probably at odds with the project's naming convention.